ament_tools/build_type_discovery.py

```diff
@@ -42,10 +42,10 @@ def get_class_for_build_type(build_type):
     entry_points = list(yield_supported_build_types(build_type))
     if len(entry_points) > 1:
         # Shouldn't happen, defensive
-        raise RuntimeError("More than one build_type entry_point.")
+        raise RuntimeError('More than one build_type entry_point.')
     if len(entry_points) == 0:
         raise MissingPluginError(
             "No plugin to handle a package with build_type '{0}'"
-            .format(build_type)
+            .format(build_type),
```

Comment:
Is this required? This doesn't make sense to me if `MissingPluginError` doesn't take any more arguments...

Reply:
We will not use the commas plugin: https://github.com/ros2/ci/pull/98#discussion_r135926777